### ISSUE TITLE
Multiple Representation of a Company

### ISSUE THREAD

**Issue by williamkapke** [nodejs/board#58]
At the Board Meeting on 2017-03-27, an issue of multiple representation of a company on the Board was discussed.

The By-Laws state:

> **Only one Member which is part of a group of Related Companies shall be entitled at one time to have a representative on the Board** or vote on matters voted on by the Members provided, however, that the (i) **TSC Director or the Individual Director, but not both, may be an employee of a Platinum Member** or one of its Related Companies, notwithstanding the fact that such Platinum Member is also represented on the Board by virtue of its right to appoint a Director under Sections 2.3(a) and 4.3(a) of these Bylaws, and (ii) **the Individual Director may be an employee of a Gold or Silver Member or one of its Related Companies**, notwithstanding the fact that such Member may already be represented on the Board by an employee elected to the Board of Directors. 

It was discovered, and confirmed with legal, that there is currently a conflict that needs to be resolved. Rod Vagg (the TSC Director) and Dan Shaw (a Gold Member Director) are both employed by NodeSource. Since NodeSource is not a Platinum Member, they do not meet the exclusion clause for Platinum Members.

We can change the By-Laws, with a 2/3 vote, to allow 2 people from the same company to be on the board. However, it is unclear if the TSC & community would support this change and it is believed this rule was to provide an incentive for a company to pay to become a Platinum member.

If the By-Laws are not changed: Rod or Dan will need to resign, assign someone in their stead, or somehow be removed.

The Board believes it is important to both notify the community of this situation and collect feedback on the possible solutions. Please comment with which solution you prefer and why. Remember that the solutions would apply to **all companies**. Feedback that specifically addresses NodeSource, or any company, in particular will be considered derailing and will be moderated. Any other comments that stray from direct feedback on possible solutions will also be considered derailing and will be moderated. If you have related questions, feel free to open up another issue on this repo! Thanks!

//cc @nodejs/tsc

**Comment by ashleygwilliams** [comment]
hi! senior individual rep here, just going to restate what i said in the Board Meeting on Monday:

i believe it's important that we maintain the representation incentive for platinum members. therefore, i would advocate that we keep the current rules as they are and either dan or rod step down. i have less opinion about exactly how we resolve the current issue of dan and rod on the board, and am more focused on the need to maintain the incentive. 

i would additionally add that given the size of the board and node community (relatively small and relatively big, respectively), i believe it is also important to restrict representation to a single director per company unless that company demonstrates strong financial support, as a platinum membership would do.

there are numerous ways to be deeply involved with node and not be a voting director. serving on the TSC, CTC, community committee, and the numerous working groups are all excellent options. i do not believe that upholding this rule unfairly restricts the actions of would-be prolific contributors, which i see as the only potential counter-argument.

<hr/>

EDIT: i am also in favor of extending this representation rule to the individual board member seats.

**Comment by jasnell** [comment]
I would prefer a bylaw change such that:

The TSC director, CommComm director, and individual member directors serve their term irrespective of their employer's member status.

Any single company may not have more than two directors on the board and that if there are two, one must either be the TSC director, CommComm director, or an individual member director.

When there are two directors from the same company on the board, and that company is not a platinum member, they either vote as a unit or the TSC/CommComm/Individual member director may select an alternate who does not work at the same company to vote in their place. If the company is a platinum member, they retain the two separate votes.

So, for example, Rod and Dan may each continue to serve their terms, but when a vote comes up, they choose to either vote as a unit or Rod may designate another TSC member to vote on behalf of the TSC.

This approach would allow the TSC/CommComm/Individual membership to pick the representative they respectively feel best represents their interests regardless of who has paid to sit around the table, while preserving fairness in the voting and the platinum incentive. It also does not introduce a punishment for companies that choose to invest in materially significant ways other than a platinum board seat.

**Comment by ashleygwilliams** [comment]
NB @jasnell there is no CommComm director, afaik.

**Comment by bnoordhuis** [comment]
Why is it acceptable for an Individual Director to be from a Gold or Silver member but not the TSC Director?  That seems arbitrary and unfair.  I could support this:

> We can change the By-Laws, with a 2/3 vote, to allow 2 people from the same company to be on the board.

I also could support curtailing membership to one company, one vote.  I'll have to think that over.

**Comment by jasnell** [comment]
Aww..  there totally should be a CommComm director, but ok

**Comment by williamkapke** [comment]
@jasnell 
> or Rod may designate another TSC member to vote on behalf of the TSC

While **I** am of the strong opinion that the TSC & Individual Directors should **represent** their constituents, the law requires the voting member to be fully responsible for their choice. It is this way because Board members are given private information that their constituents do not know. So, sometimes (but IMO rarely) this can change their vote and their constituents are unable to be told why.

I like your idea- but having a 1-off stand-ins seems like it would be risky, and maybe impossible, from a legal standpoint.

**Comment by indexzero** [comment]
@bnoordhuis 
> Why is it acceptable for an Individual Director to be from a Gold or Silver member but not the TSC Director? 

The Individual Director and the TSC Director can both be from a Gold or Silver member company. In fact, the TSC Director is from a Gold member company, and one of the Individual Directors is from a Silver member company. The issue at hand here is that (according to the by-laws) no Gold or Silver member company can have *two Directors* _of any type_ (Gold, Silver, Individual, or TSC) on the Board.

**Comment by jasnell** [comment]
It needn't be a one off. The TSC could easily vote on and select a standing alternate. In fact, we could easily designate that the TSC chair fills that role. It becomes a bit murkier for individual member reps, but perhaps it's OK that they would not have a voting alternate since there are two individual member reps.

**Comment by williamkapke** [comment]
@jasnell Indeed, the By-Laws allow the TSC Director to assign another TSC member, in writing, to act in their stead. But, that person will need to be fully onboarded and brought up to speed to be able to make informed decisions. So, yes- doable... but not at a whim. Not suggesting that's what you were saying- just nitpicking the "one off" phrase I used to paraphrase ;)

**Comment by hackygolucky** [comment]
Trying to get some clarity on @jasnell's proposal.

> 1. The TSC director, CommComm director, and individual member directors serve their term irrespective of their employer's member status.
  
Does this mean that the TSC director and the two Individual Directors could potentially all be from the same company but expected to be neutral and vote for who they represent(TSC/community)? Or is 2. below a detail that gets added on as a further condition?

> 2. Any single company may not have more than two directors on the board and that if there are two, one must either be the TSC director, CommComm director, or an individual member director.

**Comment by indexzero** [comment]
@jasnell 
> (...) they either vote as a unit or the TSC/CommComm/Individual member director may select an alternate who does not work at the same company to vote in their place.
> 
> So, for example, Rod and Dan may each continue to serve their terms, but when a vote comes up, they choose to either vote as a unit or Rod may designate another TSC member to vote on behalf of the TSC.

I'd like to point out that this completely ignores the representation that the second member is supposed to provide for the other set of members (whomever they may be). There is no point to being a full member of the Board of Directors without a vote. In this particular case if this were to move forward, there would be little incentive be for another member company to join the Foundation as Gold. Prioritizing recruitment of another Gold member company is important to rebalance the overall structure of the member companies as currently there are not even enough (minimum three) to hold a proper election for the Gold member class.

Further, in the _general case_ (remember this is _not_ about [Company Name Removed by @jasnell per moderation request] ) this puts the TSC/Individual Board member in a very awkward position. By this I mean that generally speaking the Gold/Silver/Platinum Board members join the Foundation from middle to upper management of their respective company. Conversely, the TSC/Individual Board member is more likely to be an Individual Contributor at the member company. This creates a power imbalance between the two where (behind the scenes at the member company) the Gold/Silver Board member could potentially pressure (or even directly force through management changes, etc.)  the TSC/Individual Board member to step-aside so they can retain their seat. In the extreme case consider this: _how would this play out if the Gold/Silver Board member is the direct manager of the TSC/Individual Member? _

**Comment by jasnell** [comment]
Item two is a further condition. With my proposal, we could not have more than two directors from the same company. Period. And one of those two, when it happens, must be either the TSC director or individual member director. And when it happens, they would vote as a unit, or in the case of the TSC where an alternate may be selected, the alternate may vote instead. Exception given only to platinum members who retain separate votes. 

The possible same company combinations then become: 

Platinum + Elected = 2 votes 
Gold/Silver + Individual member director = 1 vote
Gold/Silver + TSC Director = 1 vote 
Gold/Silver + TSC Alternate = 2 votes 
Individual member director + TSC director = 1 vote 
Individual member director + TSC Alternate = 2 votes

This, of course, assumes that the TSC Alternate does not work at the same employer as the TSC Director... Which is a good practice anyway.

**Comment by jasnell** [comment]
This, of course, assumes that the TSC Alternate does not work at the same employer as the TSC Director... Which is a good practice anyway.

**Comment by indexzero** [comment]
@jasnell 
> And when it happens, they would vote as a unit, or in the case of the TSC where an alternate may be selected, the alternate may vote instead.

The problem remains: the Gold/Silver director could potentially directly instruct (by virtue of their position at a given member company) the TSC/Individual Director to _select an Alternate_ or _vote as a unit._ The Platinum member class is somewhat less vulnerable to this as their Board seat is guaranteed whereas Gold/Silver/TSC/Individual are all elected.

> _EDIT: further upon more reflection this exclusion must exist for Platinum member companies. Without the exclusion, a an employee from a Platinum member company could never run for TSC/Individual Board positions._

**Comment by jasnell** [comment]
@indexzero ... just so I can understand... what are you proposing as an alternative?

**Comment by indexzero** [comment]
@jasnell I am advocating for By-laws as they are written today. In other words: I believe the current language limiting 1 person at a time is there for good reason.

**Comment by jasnell** [comment]
@indexzero.. thank you for clarifying :) that's helpful

**Comment by ashleygwilliams** [comment]
<details>
i think clarifying what people would want to happen is great as it makes the feedback we intend to bring to the board easier to collect and explain. thanks for all the feedback so far!
</details>

**Comment by nebrius** [comment]
>  Conversely, the TSC/Individual Board member is more likely to be an Individual Contributor at the member company. This creates a power imbalance between the two where (behind the scenes at the member company) the Gold/Silver Board member could potentially pressure (or even directly force through management changes, etc.) the TSC/Individual Board member to step-aside so they can retain their seat. In the extreme case consider this: _how would this play out if the Gold/Silver Board member is the direct manager of the TSC/Individual Member? _

I want to highlight this, as this is something I hadn't thought of before. I think there is merit here, although I would like to provide a bit of a counterpoint: A manager of an individual contributor who is the TSC/Individual Board member who is _not_ on the board, but still with a vested interest in what the board decides could still have motivation to apply this pressure, and there's nothing we can do in this case. I'm not saying this isn't an issue though, I do think it's an issue, but I think things do get complicated when we dig into the weeds.

🤔

**Comment by jasnell** [comment]
If the bylaws are left in place as they currently are, the issue of power-balance still occurs. For instance, a TSC Director who works for a Platinum member company could very easily be directed on how to vote on issues. Such a power imbalance should not be viewed as a "perk" of being a platinum member. If the rule that limits 1-board-member-per-company is kept in place, then it should be extended to apply equally across the board. That is, both the Platinum/TSC-Director and the Gold/Silver/Individual-member rep exceptions should be eliminated such that there really is only one representative per company on the board for all membership levels. Platinum members would still have the advantage of a guaranteed seat at the table. Gold/Silver members would be given the mutually exclusive choices to (a) Upgrade to Platinum, (b) Run for Gold/Silver seats, or (c) Encourage their employees to run for Individual member rep seats or the TSC Director seat.

That said, I'm currently sticking to my original proposal.

**Comment by williamkapke** [comment]
NOTE: This original message has been removed.

Its purpose was to state the urgency of this matter but I didn't word it in an appropriate way to convey that. (Apologies.)

**Comment by mikeal** [comment]
It's important to remember that these bylaws are from a standard boilerplate and weren't written specifically for our project or our community except in the few places that we added or amended them.

We have a very large and very strong community. We also have a very strong project now, with many contributors and people engaged in decision making. While the TSC has struggled a bit with membership as so many of its responsibilities have been delegated to other groups it is also currently growing.

The expectation of both the Individual Membership Directors and the TSC Director is that they will represent the best interests of those that elected them and not their employer. Their seat is not tied to their employer, they are free to switch employers at any time without losing their seat.

Corporate Member representatives, those elected by class and those from Platinum Members, are expected to represent the interests of their employer and potentially their class of members which includes their employer. If a director switches jobs they lose their seat.

Considering the employer of an elected at-large director conflicts with these expectations. The employer is not suppose to matter, we expect that anyone elected to these roles will put the concerns of their employer aside and represent those that elected them. If a person elected to one of these roles feels they cannot accomplish this they can recuse themselves from specific items of agenda (and, in fact, this has already happened a few times). This is pretty common in other foundations and directors should be encouraged to recuse themselves when they feel appropriate.

I'm not seeing how you reconcile the logic behind restricting membership by employer for these roles with the expectations that these people, regardless of their employer, are not representing those interests. If you believe there is inherently a conflict then there is also a conflict when that person's employer is not even on the board or even a member.

If you pull on this thread you end up with some very strange scenarios, like a company losing their seat because they hired an at large director. If instead you continue to push forward with the belief that people can, in fact, represent the interests of their voters rather than their employer you don't end up these situations. I guess what I'm saying is that there's a consistency to the rules if you go down one path and cascading inconsistency when you go down the other.

**Comment by mikeal** [comment]
<details>
  <summary>Removed Quote</summary>
  <p>There was a quote from @williamkapke here originally. The quoted comment was removed and this edit reflects the change. -wm</p>
</details>
<br>

What this means in practical terms is that both directors are still included, vote on issues, etc, until the board resolves the issue. However, because one of them may not be in good standing we only count one of them for quorum.

**Comment by indexzero** [comment]
@jasnell 
> If the bylaws are left in place as they currently are, the issue of power-balance still occurs. For instance, a TSC Director who works for a Platinum member company could very easily be directed on how to vote on issues. 

I made note of this in my edit to https://github.com/nodejs/board/issues/58#issuecomment-290136607, but might have gotten lost in the shuffle. You are definitely right about the risk, but since Platinum members are guaranteed a Board seat this exclusion must exist. If there was no such exclusion then an employee from a Platinum member company could never run for TSC/Individual Board positions. Being a Platinum member is a significant cost; saying the exclusion comes for free to Gold & Silver member classes decreases the marginal value of being Platinum and would do more harm than good.

**Comment by ashleygwilliams** [comment]
>  Being a Platinum member is a significant cost; saying the exclusion comes for free to Gold & Silver member classes decreases the marginal value of being Platinum and would do more harm than good.

would like to reiterate that the Foundation is trying to get more Platinum members and that degrading the benefits of a Platinum membership certainly does not help that situation.

<hr/>

EDIT: if it is not clear, i am agreeing strongly with @indexzero

**Comment by williamkapke** [comment]
...and convincing enough of the Current Platinum Members to vote to let go of that benefit might be difficult. If that isn't even possible, then this discussion might be moot.

Perhaps we should start with a simple vote to decide of the The Board wants to pursue changing the By-Laws to expand the Multiple Representation rules at all.

**Comment by ashleygwilliams** [comment]
that's a great idea @williamkapke, i think discussing it with the community is still a good idea either way.

**Comment by jasnell** [comment]
Presumably the goal is to grow membership at *every* level, not *only* at the Platinum level. It also should be recognized that the *proportionate* cost to a small Gold/Silver-level company that is investing heavily on technical contributions could be significantly higher than that of a Platinum member.

That said, my proposal *does not* degrade the benefits of Platinum membership. Under my proposal, Platinum members would *still* have a guaranteed seat at the table and would *still* be guaranteed two distinct votes in the case a Platinum member employee happens to be voted in as one of the "At Large" Director positions. In other words, there is zero change for Platinum members under my proposal. Platinum members would be giving up *nothing*.

The potential power imbalance that @indexzero describes *is already present* in the current bylaws -- you just have to have lots of money available to buy the right to take advantage of it.  The point here, however, is that my proposal does not introduce a *new* problem into the mix, and as @mikeal points out, At Large Directors are fully expected to act on behalf of their constituents and not their employers; nor are their terms tied to their employers in any way.

And as has been already noted, the bylaws already allow for the TSC Director to nominate an alternate to act in their place. My proposal would fortify that and more clearly spell out how that works, including introducing a requirement that the TSC alternate not work for the same employer as the TSC director.

The tl;dr version is this: my proposal does not take *anything* from Platinum members. It *does* place a restriction on Gold/Silver member companies (the single vote requirement). It *does* allow the TSC and individual members to pick whichever individual they feel best represents their interest. And it *does not* introduce any *new* issues into the mix. (Tho it admittedly does not solve the potential power imbalance problem).

*If* the intent is to actually address the power-imbalance problem, the only way to do so is to remove the ability for *any* company to have more than one employee on the board, including Platinum members.

In my opinion, it's far too early to start calling for any kind of vote. We need more input from a broader range of voices.

/cc @nodejs/tsc @nodejs/ctc

**Comment by ashleygwilliams** [comment]
@jasnell the only reason that @williamkapke mentioned a vote is because it is possible that the board won't allow any changes to be made at all. the board could block this even if the community disagreed. that's all! i don't think the board *would* do that, they care about the community input, but that's really all that Will meant.

**Comment by mikeal** [comment]
> @jasnell 
>  The point here, however, is that my proposal does not introduce a new problem into the mix.

I think it actually does.

To my mind, the current issue with the bylaws is that they simultaneously **1)** ask at-large directors to represent their constituency irrespective of employer and **2)** contain language that imposes specific restrictions based on the employer they are required to refrain from representing.

This extends that problem by adding even more language and rules related to the employer of at-large directors.

We should be consistent in our expectations of board directors and the rules by which we ask them to operate under.

What seems most prudent is to take the language we have for Individual Directors that makes it clear they are not representatives of their employer and exempts them from the associate rules, and extend it to the elected TSC Director.

**Comment by ashleygwilliams** [comment]
@mikeal can you please @-mention the person you are quoting so it's easier to follow?

**Comment by mikeal** [comment]
@ashleygwilliams  edited :)

**Comment by ashleygwilliams** [comment]
thanks!

Additionally, @mikeal just to make what is on the table here clear, i think we'd like to align the rules for Individual Membership Directors and the TSC director. Which means that we might adopt the language for the TSC director and also apply it to Individual Directors. Either way would be an improvement in consistency. Which is better for the Board is the question at hand, in my opinion. Like most things, I don't think there is an obvious choice, as the continued convo here would confirm.

**Comment by mikeal** [comment]
@ashleygwilliams I understand, but I'm less concerned with the inconsistency between Individual Directors and the TSC Director than I am the inconsistencies with regard to the recognition of the employer of at-large directors.

I think this is pandora's box. We either elect people to these roles that we expect to represent our interests regardless of where they are employed or we don't. If we think that people elected to these roles are just representing their company then why have them at all? I don't think there is any room to add "just a little" recognition without introducing a bunch of incentive changes that we can't entirely predict.

**Comment by jasnell** [comment]
@mikeal ... what you describe isn't my proposal *adding* a new problem, it's *extending an existing one* ;-) I think I would be ok with language that extends the current exemption that exists for Individual Directors to the TSC director, as you suggest. I'm just not sure that entirely addresses the problem. It would be a good starting point, however, and I could get behind that approach. 

What I do *not* feel is a good approach is keeping and enforcing the bylaws as they are currently written; and most immediately, I would *not* support an option that requires either Dan or Rod to resign their current roles.

**Comment by mikeal** [comment]
One thing I'll add. It takes a while for new board members to ramp up and be effective. Rules that introduce sudden changes in the representative or makeup of the board mid-term cause instability that is costly. Whenever possible, it ads to the stability and effectiveness of the board to avoid changes in directors mid-term.

**Comment by indexzero** [comment]
@mikeal 

> One thing I'll add. It takes a while for new board members to ramp up and be effective. Rules that introduce sudden changes in the representative or makeup of the board mid-term cause instability that is costly. Whenever possible, it ads to the stability and effectiveness of the board to avoid changes in directors mid-term.

In general I agree with this sentiment, but it does not apply in this case. Both Dan and I were Board members before the coin toss in December that decided the current Gold Director. If the by-laws were to remain the same and he were to step down to comply with them there is no ramp up time necessary for me.

**Comment by mikeal** [comment]
Here's one other solution I'll put out there.

If an employer is already on the board and someone is elected at-large from the same company it's fair to say that the voters were confident the person could represent them regardless.

If our primary concern is that this level of confidence **changes** when either **1)** the director switches employers or **2)** their existing employer gains a corporate member seat, then we could solve that with a vote of confidence by the existing at-large voters.

Essentially, you could add language that targets these changes regarding employers and have it trigger a vote of either the Individual Membership or the TSC. The vote would simply be to either 1) affirm or 2) recall. A recall would trigger a new election for the remainder of the term.

This would leave the corporate member seats un-effected. This would not **force** a board director change except in the case that their class actually does wish to recall them.

This is obviously a lot easier for the TSC than the Individual Membership but it could work if this is really our primary concern.

**Comment by mikeal** [comment]
> In general I agree with this sentiment, but it does not apply in this case. Both Dan and I were Board members before the coin toss in December that decided the current Gold Director. If the by-laws were to remain the same and he were to step down to comply with them there is no ramp up time necessary for me.
> @indexzero 

Right, I'm not concerned about the ramp up time in this very specific case we have today but we're going to be living with these bylaws for a long time and we need to keep the long term effects in mind. The next time this happens we are unlikely to be in the same position of having a single person eligible for the seat that happens to have served on the board previously :)

**Comment by indexzero** [comment]
> Here's one other solution I'll put out there.
> 
> If an employer is already on the board and someone is elected at-large from the same company it's fair to say that the voters were confident the person could represent them regardless.
> 
> If our primary concern is that this level of confidence changes when either 1) the director switches employers or 2) their existing employer gains a corporate member seat, then we could solve that with a vote of confidence by the existing at-large voters. @mikeal

While this solution meets a number of criteria this group has set down, I would like to highlight that the point of view folks commenting on this thread has is very different than the rest of the community. If I was on the outside looking in, I could easily see this as the Node Foundation  changing the rules to accommodate our perceived best friends. This potential is troubling since this is not a message we should be sending to both existing or potential individual members and member companies.

**Comment by mikeal** [comment]
> While this solution meets a number of criteria this group has set down, I would like to highlight that the point of view folks commenting on this thread has is very different than the rest of the community. If I was on the outside looking in, I could easily see this as the Node Foundation changing the rules to accommodate our perceived best friends. This potential is troubling since this is not a message we should be sending to both existing or potential individual members and member companies.

What "best friends" benefit from an affirm/recall vote? The people it would seem to benefit are the people who voted this person in, giving them the opportunity to remove them if they have lost confidence or affirm that they are still confident in the director if they so choose.

**Comment by indexzero** [comment]
> What "best friends" benefit from an affirm/recall vote? @mikeal 

This discussion (and any potential by-law changes that come from it) are the result of a mistake by the NF. That's simply a fact. If everyone here really cares about getting this right for the sake Node Foundation itself and not those currently affected then it would be prudent to make no by-law changes, resolve the current conflict, and re-open this discussion in, say, one quarter (3 months).

This demonstrates that the issue is really important to the Board, the membership, and the TSC as a core issue. Not simply as a reaction to a mistake.

**Comment by mikeal** [comment]
@indexzero I think I see where you're coming from but there's one assumption I think you have that may not be true: it is not a given that the Gold Director would step down. Depending on the interpretation the TSC Director could be asked to step down. That's why the board wanted to get community, and TSC, input on this before they acted to either force a change or alter the bylaws.

**Comment by jasnell** [comment]
(First sentence of this comment elided by author per request). I'm quite certain that we all care about getting this right for the sake of the Foundation and the project, we just have differing opinions on how to get there and differing opinions on what the prudent next steps should be. Having an open, thoughtful, and reasoned discussion on this issue also demonstrates that the issue is really important.

For me, the question comes down to original intent. Specifically, the original intent was for the TSC to be free to pick the individual that the TSC feels best represents it's interests on Board, irrespective of that individual's employment. That intent should be protected and accounted for. As @mikeal notes, it currently is not

**Comment by indexzero** [comment]
> (First sentence of this comment elided by author per request). I'm quite certain that we all care about getting this right for the sake of the Foundation and the project, we just have differing opinions on how to get there and differing opinions on what the prudent next steps should be. Having an open, thoughtful, and reasoned discussion on this issue also demonstrates that the issue is really important. @jasnell 

I don't believe I claimed that perception was accurate, only that someone with imperfect information could have that perception. Not every community member, potential Individual Member, or potential Member Company is as in touch with the community as we. I have no objection to open, thoughtful, and reasoned discussion on this or any other issue. What I object to is urgency due to the ambiguous state introduced by a mistake. Mistakes happen, nothing is perfect. I don't say "mistake" to point fingers or blame anyone. Merely that this was missed in Dececember and now this discussion is happening as a result of it. Being reactionary to a mistake sets the precedent of being reactionary to similar events in the future. Why not rollback the stack of procedure (outlined below) and update the by-laws as a separate action? 

> I think you have that may not be true: it is not a given than the Gold Director would step down. @mikeal 

Procedurally I do not see how that is under consideration. The updated December by-laws state: _"In the event that there are two Gold Members, each desiring to vote for its own nominee, the winner shall be determined by a coin toss."_ Based on this new information the nomination itself was invalid and as such any resulting changes are also invalid. 

> For me, the question comes down to original intent. Specifically, the original intent was for the TSC to be free to pick the individual that the TSC feels best represents it's interests on Board, irrespective of that individual's employment. That intent should be protected and accounted for. @jasnell

Agreed. That the invalid nomination would potentially change the TSC Director is in direct contradiction to the original intent of the TSC. If the Board & the Foundation is to honor that original intent whatsoever having the Board ask the TSC Director to step-down should be off the table. If, however, the TSC wanted to consider a new Director as a result of this that _(of course)_ is their own decision to make under their self-governance.

**Comment by rvagg** [comment]
> @indexzero: Based on this new information the nomination itself was invalid and as such any resulting changes are also invalid.

Not necessarily @indexzero. ~~I know~~ _(ed by @rvagg, apologies for stating that I know motivations)_ if you're viewing this as an opportunity to regain the board seat by making the case that the whole _stack of procedure_ is invalid but the likely outcome here does not involve [Company Name Removed by @jasnell per moderation request] stepping down and handing over the Gold seat to [Company Name Removed by @jasnell per moderation request]. It's much more likely that I'd be stepping down from the board if we don't agree on an alternative way forward. Even rolling back to December, the coin toss could still be held and [Company Name Removed by @jasnell per moderation request] be forced to make the decision if it won. Without specific language in the by-laws, you're going to have a hard time making the case that the coin toss itself was problematic. So how about we take that off the table and try and discuss this as more disinterested parties looking to what's in the best interests of the Foundation and the project into the future _regardless_ of individuals and companies involved. I'm certainly not going to be around forever but I do care that those that come after us have something that's set up right for Node.

**Comment by ashleygwilliams** [comment]
<details>
@rvagg 

>  I know you're viewing this as an opportunity to regain the board seat by making the case that the whole stack of procedure is invalid but the likely outcome here does not involve NodeSource stepping down and handing over the Gold seat to GoDaddy.

it was asked that comments about specific companies not be made.  comments like this are counter productive and not helpful to moving the convo forward. would you mind editing your statement please?
</details>

**Comment by rvagg** [comment]
<details>
<summary><i>comment compacted in order to get things back on track - @rvagg</i></summary>
I'm pretty sure we've already been derailed with @indexzero arguing that we go back and argue the toss again. If you want to be consistent in this moderation then you should probably roll back up and remove comments about the NodeSource/GoDaddy toss.
I'm really not sure what you're trying to achieve here @ashleygwilliams, these are specific questions about specific circumstances that we can't address without being specific. You might have standing if you want to argue that @indexzero is attempting to detail by making this a question about the gold coin toss rather than the original question but you're clearly not doing that.

I don't support your proposed adjustment to the language in my comment and may have to take it directly to the board for adjudication if you take it upon yourself to do so.
</detail>

**Comment by ashleygwilliams** [comment]
<details>
  <summary>This comment not needed any further</summary>
  <p>
@rvagg stating someone else's intentions for them is never good form. your comment attacked @indexzero unnecessarily. re-reading all of the previous comments, no member companies have been attacked in any way. i urge you to please moderate your comment to be more neutral.
</p>
</details>

**Comment by indexzero** [comment]
> I know (ed by @rvagg, apologies for stating that I know motivations) if you're viewing this as an opportunity to regain the board seat by making the case that the whole stack of procedure is invalid but the likely outcome here does not involve [Company Name Removed by @jasnell per moderation request] stepping down and handing over the Gold seat to [Company Name Removed by @jasnell per moderation request] . @rvagg 

I actually don't take offense to the use of company names here @ashleygwilliams. Where we work is simply a fact. I'm sure some folks will view my comments through an opportunistic lens, but that is not my motivation. The coin toss in December itself was fair, and despite no longer being a member of the Board I have still been lobbying other potential member companies to join _as Gold_ to achieve our stated goals: growing the membership of all classes. 

> Even rolling back to December, the coin toss could still be held and [Company Name Removed by @jasnell per moderation request] be forced to make the decision if it won. Without specific language in the by-laws, you're going to have a hard time making the case that the coin toss itself was problematic. @rvagg 

That's a very good point. I had not considered that as a hypothetical re-playing of history with this new information. This is a great example of the kind of imperfect information I was referring to when responding to @jasnell earlier. Several folks on this thread had stated that either Director position could be changed _without any explanation as to why._ This explanation makes perfect sense and gives context to those previous statements.

> So how about we take that off the table and try and discuss this as more disinterested parties looking to what's in the best interests of the Foundation and the project into the future regardless of individuals and companies involved.

I believe I have approached this in such a fashion. My position is that the by-laws as-is are the correct path forward. This is because there is a natural power imbalance that exists between any Gold/Silver/Platinum Director and any TSC/Individual Director at the same company. An exception exists for Platinum members to enable employees of Platinum members to potentially serve as TSC/Individual Directors (a luxury that Gold/Silver members enjoy by definition). Who resigns is largely irrelevant to me; I'm enjoying more free time since leaving the Board 😸

**Comment by jasnell** [comment]
<details>
  <summary>Comment not needed any further</summary>
  <p>@rvagg and @ashleygwilliams : Please defer discussion of potential moderation actions to the moderation repo [thread](https://github.com/nodejs/moderation/issues/82) as they are derailing for the main conversation here. We have a [policy](https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md#collaborator-posts) for handling moderation disputes among collaborators that should be followed.</p>
</details>

**Comment by mikeal** [comment]
<details>
  <summary>Comment not needed any further</summary>
  <p>
I'm going to suggest that we cut off this specific thread as it has escalated to a point many of us are no longer comfortable with.

Any objections to closing/locking this issue and logging a few more **specific** issues about the proposed resolutions?</p>
</details>

**Comment by williamkapke** [comment]
<details>
  <summary>Comment not needed any further</summary>
  <p>
@mikeal yes, I am not able to stay up with the fast pace messages happening here and it seems to have gotten a bit overloaded (to say the least). I have locked it for now.</p>
</details>

**Comment by rvagg** [comment]
So, to change tack a little bit I've been talking with TSC folks about the implications for the technical side of the Foundation as they stand different from the Board. We've talked here already about _incentives_ for Foundation finances but the TSC primarily cares about a different set of incentives.

Consider the following:

* The technical team needs developers to be involved, while we are trying hard to diversify that involvement and make the long tail of casual contributors _fatter_, it will be the case for the foreseeable future that the largest amount of the work will be done by developers who are paid to contribute to Node, either part-time or full-time.
* It is probable (not unlikely) that the TSC Director will be someone who is in such a paid position.
* Companies that pay for Node development will tend to be either large corporations, such as those we find at Platinum level today, or mid-tier companies that have a direct interest in Node, consider [Company Name Removed by @jasnell per moderation request] and [Company Name Removed by @jasnell per moderation request] as examples of this.
* Platinum membership is super-expensive and very difficult for mid-tier companies to justify. It's possible that we will see a [Company Name Removed by @jasnell per moderation request] or [Company Name Removed by @jasnell per moderation request] at Platinum one day but even when we do, we know it'll be a stretch.

I'm going to use [Company Name Removed by @jasnell per moderation request] as an example here because it's the most obvious and I have the most information about how it works, but I suspect that you could apply a very similar lens to [Company Name Removed by @jasnell per moderation request] and a few others who invest in core.

The biggest impact you can have around the open Node.js ecosystem without paying for a Platinum seat is to invest in development of core and its surrounds. That's what [Company Name Removed by @jasnell per moderation request] has done, it has 3 individuals who's primary responsibility has been to contribute to core. That connects [Company Name Removed by @jasnell per moderation request] deeply to the project and gives it a seat at some of the relevant tables that matter to it (LTS being a great example). If [Company Name Removed by @jasnell per moderation request] were to opt for a Platinum seat then it'd have to sacrifice a large portion of the developer spend that it currently makes in exchange for a relatively symbolic position.

So, from the TSC's perspective, this specific question pits incentives against each other. One is toward Platinum, and another is toward investment in technical effort in core. So it's not clear cut at all!

Now consider the **individuals** involved here. A TSC Director will either have their employment opportunities reduced because they may cause conflict for the company(ies) involved or be prepared to step down as soon as they come under a different banner. This also puts Platinum companies (i.e. large companies) at a much better talent acquisition position than the mid-tier companies because they have the extra seat.

To be clear - I'm not arguing for a particular outcome on this, but please let's keep some perspective when talking about incentives because there's more than one direction involved.

**Comment by jasnell** [comment]
@williamkapke ... locking before anyone else has had the opportunity to comment is a bit unfair. I disagree that there's a need to lock the thread. But, it's done, so very well. (update: by "anyone else" I mean specifically collaborators, and by "comment" I mean weigh in on whether locking is necessary... sorry if that wasn't clear)

There were several important things discussed in this thread that cannot be lost. Based on the conversation, we have:

1. The Foundation Board is currently in violation of it's own established bylaws and, accordingly, appears to have selected a Gold member director that should not be eligible per the by laws.

2. There is a question about whether the bylaws should be changed.

3. However, before we can discuss whether the bylaws should be changed, a decision needs to be made about the current violation, specifically: does it need to be unwound and corrected before the bylaws change can be discussed.

While it's not my personal preference, I understand what @indexzero is arguing here with regards to unwinding the stack to correct the violation. It makes sense even if I don't like the result (specifically, that has nothing to do with the specific people involved, I'd just rather the currently elected board members to continue in their current terms until the next natural transition cycle). But, @indexzero makes a good (albeit admittedly somewhat self-serving) case here that cannot be ignored.

Even so, I would *strongly* argue that the bylaws need to be changed. The TSC Director should have the same employer exemptions as the Individual Member directors. This is ultimately what I want to see happen coming out of this conversation.

**Comment by mikeal** [comment]
> However, before we can discuss whether the bylaws should be changed, a decision needs to be made about the current violation, specifically: does it need to be unwound and corrected before the bylaws change can be discussed.

The board asked for direction from the community and TSC into how to proceed. They elected to leave things as is, acknowledging that one director is not in good standing and to take that into account during quorum counts.

Basically, the preferred resolution should be framed as either a specific enforcement of the current bylaws or an adjustment. You don't need to draft the bylaw change, we have lawyers for that, but the recommendation needs to be laid out.

**Comment by ashleygwilliams** [comment]
<details>
this thread was created so that @williamkapke and i could get feedback. that got out of hand. as the senior individual member rep and co owner of this repo i am closing this until we can recap and reorganize. i'll let you know when we have a better strategy for this.

please cease commenting on the thread or open your own on another repo.
</details>

**Comment by williamkapke** [comment]
Thanks for the input today everyone. I've had a chance to digest the comments now.

@jasnell & @mikeal , you have a proposal for how to change the by-laws. It would be great if each of you could open a new issue and lay out the proposal.

**Comment by ashleygwilliams** [comment]
@williamkapke i disagree. i'd like to get a handle on what the board is thinking as well as some moderation plan before we continue. this way we can have a better chance of success in the other issues. @mikeal and @jasnell if you could hold off, i would appreciate. thanks!

**Comment by mikeal** [comment]
@ashleygwilliams will do. let me know when and how you'd like to proceed once you and @williamkapke are in alignment on how to moderate it.
